RoboJackets/robocup-software
# Basic Offensive Scramble Play

## Description
A play for the offensive scramble situation. One robot captures ball, another one moves behind that one and another to the left or right depending on the position of the ball

## Associated Issue
Issue #1347 

## Steps to test
Just run it in sim with situational analysis to see if it works

Expected result: It should hopefully get the ball and move into an attack goal play

## Questions:
Are the distances for the cross and dropback points good?
